src/SqlSquared/Signature.purs

```diff
@@ -138,8 +142,20 @@ data SqlF literal a
   | Select (SelectR a)
   | Parens a
 
+type AnyDirPath = E.Either (Pt.AbsDir Pt.Unsandboxed) (Pt.RelDir Pt.Unsandboxed)
+
+printAnyDirPath :: AnyDirPath -> String
+printAnyDirPath = E.either Pt.unsafePrintPath Pt.unsafePrintPath
+
+parseAnyDirPath :: forall m. Applicative m => (forall a. String -> m a) -> String -> m AnyDirPath
```

Comment:
Can't we move `a` to the left?

Reply:
We should be able to do this
```
forall m. (forall a. String -> a) -> any ~ 
forall m. (exists a. (String -> a) -> any) ~ 
forall m a. (String -> a) -> any
```

Reply:
I moved a to top forall but it fails with:
```
[1/1 TypesDoNotUnify] src/SqlSquared/Path.purs:41:4

  41    (pure ∘ E.Right)
         ^^^^^^^^^^^^^^
  
  Could not match type
  
    Either t4 (Path Rel File Unsandboxed)
  
  with type
  
    a1
  
  while trying to match type t2 t3
    with type m0 a1
  while checking that expression (compose pure) Right
    has type Path Rel File Unsandboxed -> m0 a1
  in value declaration parseAnyFilePath
  
  where a1 is a rigid type variable
        m0 is a rigid type variable
        t3 is an unknown type
        t2 is an unknown type
        t4 is an unknown type
```
so reverting the last change